samza-core/src/main/scala/org/apache/samza/config/TaskConfig.scala

```diff
@@ -96,11 +96,6 @@ class TaskConfig(config: Config) extends ScalaMapConfig(config) with Logging {
     case _ => TaskConfig.DEFAULT_COMMIT_MS
   }
 
-  def getShutdownMs: Option[Long] = getOption(TaskConfig.SHUTDOWN_MS) match {
```

Comment:
What happened to this? Present in the java version?

Reply:
This was duplicated between TaskConfig.scala and TaskConfigJava.java 
(there is duplication between these, Im working parallely on removing)